SUPPORT.md

```diff
@@ -14,6 +14,22 @@ You can find support via:
 - [Slack](https://silverstripe.org/slack)
 - [Stack Overflow (remember to tag as silverstripe)](http://stackoverflow.com/questions/tagged/silverstripe)
```

Comment:
Maybe the bullet points should be reordered in priority.

Reply:
Thanks for the suggestion. I've updated the PR.